### PR TITLE
Fix OvenPlayer fetch null

### PR DESCRIPTION
**Pull request by dygy**
https://github.com/AirenSoft/OvenPlayer/issues/374

### CODE REVIEW

**Comment by dygy** [comment]
@SangwonOh

File: src/js/api/media/Manager.js [hunk 1]
```diff
@@ -65,8 +65,6 @@ const Manager = function(container, browserInfo){
     that.destroy = () =>{
         $container.removeChild();
         $container = null;
-        videoElement.src = null;
```
Comment (SangwonOh) [inline src/js/api/media/Manager.js:68]: These codes were added due to the following issue.
I think We need to test to see if this issue doesn't occur.

https://github.com/AirenSoft/OvenPlayer/issues/361
https://github.com/AirenSoft/OvenPlayer/commit/00581a88393c9dbd44a84189c0a84b3ee52d354f

**Review by SangwonOh — COMMENTED** [review]
I think testing is needed.

File: src/js/api/media/Manager.js (hunk 1 above)
Comment (dygy) [reply src/js/api/media/Manager.js:68]: > These codes were added due to the following issue. I think We need to test to see if this issue doesn't occur.
> 
> #361 [00581a8](https://github.com/AirenSoft/OvenPlayer/commit/00581a88393c9dbd44a84189c0a84b3ee52d354f)

@SangwonOh i've tested on safari, srcObject is required and does not gave us fetch /null

I've returned it

**Review by SangwonOh — APPROVED** [review]
Thanks very much! 👍

**Pull request merged**